<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wifiSetup</class>
 <widget class="QDialog" name="wifiSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="wifiConnect">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wifiRefresh">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wifiCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wifiPassword">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Password...</string>
   </property>
  </widget>
  <widget class="QListWidget" name="wifiList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>321</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>wifiList</tabstop>
  <tabstop>wifiPassword</tabstop>
  <tabstop>wifiConnect</tabstop>
  <tabstop>wifiRefresh</tabstop>
  <tabstop>wifiCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
